src/drivers/mct/cime_config/config_component.xml

```diff
@@ -11,7 +11,7 @@
 
   <entry id="COMP_CLASSES">
     <type>char</type>
-    <default_value>CPL,ATM,LND,ICE,OCN,ROF,GLC,WAV,ESP</default_value>
+    <default_value>CPL,ATM,LND,ICE,OCN,ROF,GLC,WAV,IAC,ESP</default_value>
```

Comment:
Please move this field to config_component_e3sm.xml and config_component_cesm.xml and remove the changes from config/cesm/*

Reply:
Are you sure that is the right thing to do?  The system will have to build a stub IAC and link against it because of the coupler so might as well maintain a common COMP_CLASSES.

Reply:
We agreed that IAC field should remain in the shared coupler XML file.